indico/modules/events/surveys/templates/management/survey_submission.html

```diff
@@ -21,13 +21,26 @@
 {% endblock %}
 
 {% block content %}
-    <div class="i-box-content">
-        <dl class="i-data-list">
-            {% for answer in submission.answers -%}
-                <dt><strong>{{ answer.question.title }}</strong></dt>
-                <dd>{{ _render_answer(answer) }}</dd>
-            {%- endfor %}
-        </dl>
+    <div id="survey-submission" class="i-box-content">
+        {% for section in survey.sections if section.children -%}
```

Comment:
This is a `SurveySubmission` object, there are no `sections` in it. I tried `joinedload('survey').defaultload('sections').joinedload('children')`, but it doesn't seem to work either :disappointed:.